instrumentation-genai/opentelemetry-instrumentation-openai-v2/examples/manual/main.py

```diff
@@ -24,6 +34,66 @@
     BatchSpanProcessor(OTLPSpanExporter())
 )
 
+# configure metrics
+reader = PeriodicExportingMetricReader(
+    OTLPMetricExporter(), export_interval_millis=1000
+)
+
+# Configure metric views that allow to customize the bucket boundaries for histogram metrics.
+# This should not be necessary in the future - https://github.com/open-telemetry/opentelemetry-python-contrib/issues/3235.
+# The bucket boundaries ensure that the metrics are correctly aggregated and displayed in the backend.
+#
+# The bucket boundaries are defined as follows:
+# For `gen_ai.client.token.usage`: [1, 4, 16, 64, 256, 1024, 4096, 16384, 65536, 262144, 1048576, 4194304, 16777216, 67108864]
+# For `gen_ai.client.operation.duration`: [0.01, 0.02, 0.04, 0.08, 0.16, 0.32, 0.64, 1.28, 2.56, 5.12, 10.24, 20.48, 40.96, 81.92]
+#
+# If your application benefits from different bucket boundaries, you can update them as needed.
+views = [
+    View(
+        instrument_name="gen_ai.client.token.usage",
+        aggregation=ExplicitBucketHistogramAggregation(
```

Comment:
IIUC what we are doing is avoiding the default boundaries being used both for token usage and also duration.

From [here](https://github.com/open-telemetry/opentelemetry-python/blob/4a271f946a23060bf99f7d6c0aeaad63cd8c764d/opentelemetry-sdk/src/opentelemetry/sdk/metrics/_internal/aggregation.py#L448-L464)
```python
_DEFAULT_EXPLICIT_BUCKET_HISTOGRAM_AGGREGATION_BOUNDARIES: Sequence[float] = (
    0.0,
    5.0,
    10.0,
    25.0,
    50.0,
    75.0,
    100.0,
    250.0,
    500.0,
    750.0,
    1000.0,
    2500.0,
    5000.0,
    7500.0,
    10000.0,
)
```

So, this default might be ok for tokens for some apps, but probably not reasonable for durations which are in seconds (except maybe in reasoning models ;)). Correct me if I'm wrong!